Doc/c-api/init.rst

```diff
@@ -622,7 +622,8 @@ Process-wide parameters
       It now returns ``NULL`` if called before :c:func:`Py_Initialize`.
 
    .. deprecated-removed:: 3.13 3.15
-      Get :data:`sys.executable` instead.
+      Use ``PySys_GetObject("executable")`` (:data:`sys.executable`) or
```

Comment:
I think that `PyConfig_Get("executable")` is more preferable in a new code, isn't it?

`PySys_GetObject()` will be deprecated in future (it returns a borrowed reference and silences errors).

Reply:
PySys_GetObject() works on all Python versions, whereas PyConfig_Get() is new in Python 3.14. The note can be updated once PySys_GetObject() will be deprecated.

Reply:
> PySys_GetObject() works on all Python versions, whereas PyConfig_Get() is new in Python 3.14.

I don't understand; this PR is for the 3.14 docs, so it should be ok to mention new-in-3.14 APIs here, no?

Reply:
If possible, I would prefer to provide a replacement working on old Python versions as well.

Reply:
I updated the PR to use only `PyConfig_Get()`.